pyracing/response_objects/iracing_data.py

```diff
@@ -28,3 +28,48 @@ def value_or_none(dict, field_name):
             return dict[field_name]
         else:
             return None
+
+
+class CarClass:
+    def __init__(self, dict):
+        self.rel_speed = dict['relspeed']  # Speed ranking to other classes
+        self.lower_name = dict['lowername']
+        self.custid = dict['custid']
```

Comment:
Does this actually have a cust_id? Because that is incredibly confusing

Reply:
yup, copied directly from returned data. Most of them return zero as the value, but if you do something like carclassid:150, it returns a custid of 279146

Reply:
I have no clue what that even means then lol but thats fine, we can model it